src/MPGDDIRC_geo.cpp

```diff
@@ -123,13 +123,13 @@ static Ref_t create_MPGDDIRC_geo(Detector& description, xml_h e, SensitiveDetect
         max_component_length = box_length;
         gas_thickness += x_comp.thickness();
         c_box = {box_width / 2, box_length / 2, x_comp.thickness() / 2};
-        printout(DEBUG, "MPGDDIRC_geo", "gas: %f", comp_name);
+        printout(DEBUG, "MPGDDIRC_geo", "gas: %s", comp_name.c_str());
```

Comment:
Hmm. it seems like our CI should have caught this... Maybe gcc vs clang? Suggestions on how to avoid?

Reply:
This was using clang

Reply:
#215 should hopefully avoid this